walkthroughs/week-3-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java

```diff
@@ -15,7 +15,7 @@
 package com.google.sps.servlets;
 
 import com.google.sps.data.ServerStats;
-import com.google.gson.Gson;
+'import com.google.gson.Gson;'
```

Comment:
Was this change intentional?

Reply:
It was not -- I think it was a typo on my end.